experiments/standalone/Soil/evaporation.jl

```diff
@@ -211,60 +214,62 @@ for (FT, tf) in ((Float32, 2 * dt), (Float64, tf))
             S_l_sfc = ClimaLSM.Domains.top_center_to_surface(
                 effective_saturation.(ν, u.soil.ϑ_l, θ_r),
             )
-            layer_thickness =
-                parent(Soil.dry_soil_layer_thickness.(S_l_sfc, S_c, d_ds))[1]
-            push!(dsl, layer_thickness)
+            layer_thickness = Soil.dry_soil_layer_thickness.(S_l_sfc, S_c, d_ds)
+            push!(dsl, parent(layer_thickness)[1])
```

Comment:
I allocated the space up front, but am not sure about using `parent`. In the end, we want to plot these quantities, and Im not sure about plotting a timeseries of PointFields (vs a timeseries of Floats).

let me know what you think!

Reply:
Ahh yes I think it's fine since we'll need to access the parent for plotting. I was just thinking that we want to avoid using `parent` to encourage GPU compatibility, but I think this is okay